Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1970,13 +1970,12 @@ ErrorOr<void> Element::scroll_into_view(Optional<Variant<bool, ScrollIntoViewOpt
 
 void Element::invalidate_style_after_attribute_change(FlyString const& attribute_name, Optional<String> const& old_value, Optional<String> const& new_value)
 {
+    Vector<CSS::InvalidationSet::Property> changed_properties;
```

Comment:
Maybe we can prevent allocation by giving it an inline size, since we know it to always contain at least one changed attribute?